docs/guides/data-binding/how-to-bind-tabs.md

```diff
@@ -49,7 +49,7 @@ Finally create a `TabControl` and bind its `ItemsSource` property to the DataCon
       <DataTemplate>
```

Comment:
here ^

Reply:
Sorry for beginner's confusion with pull requests. I made three pull requests for three branches on my local repository. The third one  should be right.

I got confused because, when I tried to make a correction and pull request, GitHub asked me whether I wanted to make a new branch and pull request.  I wondered whether, if I did not make a new branch, my change would not be in a pull request, so I did it to be safe.  I'm now fairly sure that was unnecessary, as GitHub had previously told me something like I can just update my original fork branch to make a pull request? (Or add to an existing pull request?)

Reply:
No need for new branches besides the original PR branch. Work in the same branch and make changes. What I do is commit in IDE without pushing, hop into CLI, and `git push -f origin HEAD`. The new commit will automatically propagate into your PR and be shown here.

Reply:
@stevemonaco, thanks for the tip. My workflow is a little different. I never commit in the IDE.  I use GitKraken for all things Git.  I wonder what would happen if I were to pushed from GitKraken rather than the Git CLI.

Reply:
I haven't used GitKraken, so I don't know.

Reply:
OK, I've successfully updated this pull request with the change @timunie suggested. (@stevemonaco, pushing from GitKraken works fine.)

I've deleted the other two branches I unnecessarily created.  For unknown reason, before I did that, I could not see pull requests for those branches.  As I will never do that again, I can put it down to one of those unexplained mysteries.  If those pull requests never existed, that would explain why they elicited no response.